src/ui/Island/RealmDetails/Quests/XPCountdown.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react"
+import {
+  selectSeasonDurationInWeeks,
+  selectSeasonWeek,
+  selectWeekEndDate,
+  useDappSelector,
+} from "redux-state"
+import { getTimeRemaining } from "shared/utils"
+
+export default function XPCountdown() {
+  const weekEndDate = useDappSelector(selectWeekEndDate)
```

Comment:
This is actually not correct at all because weeks here are changing on a different time than our XP drops... we can assume we are doing the drops every thursday on a given time instead. Or we can calculate what is the delta time between our manual drop time and the date we have here.